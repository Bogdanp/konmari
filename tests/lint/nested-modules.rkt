#lang racket/base

(module+ test
  (provide a)

  (define b 1)
  (define b 1)

  (if 1
      (let ()
        2)
      (begin
        3)))
