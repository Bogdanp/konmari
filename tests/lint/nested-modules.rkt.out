/nested-modules.rkt:7:11:error:identifier 'b' is already defined
/nested-modules.rkt:9:3:warning:use a cond expression instead of nesting begin or let inside an if
/nested-modules.rkt:18:11:warning:identifier 'x' shadows an earlier binding
/nested-modules.rkt:4:12:error:identifier 'a' provided but not defined
